frontend/src/console/components/CoursesManage/CoursesManage.tsx

```diff
@@ -0,0 +1,127 @@
+  public render() {
+    let instanceLink;
+    if (this.props.activeInstance.data) {
+      instanceLink = this.props.activeInstance.data.studioUrl;
+    } else {
+      instanceLink = ROUTES.Error.UNKNOWN_ERROR;
+    }
+
+    return (
+      <ConsolePage contentLoading={this.props.loading}>
+        <div className="courses-page">
+          <Container fluid className="container">
+            <ConsolePageCustomizationContainer>
+              <Row>
+                <Col md={9}>
+                  <h2>
+                    <WrappedMessage id="title" messages={messages} />
+                  </h2>
+                  <WrappedMessage id="paragraph_01" messages={messages} />
+                </Col>
+              </Row>
+              <Row className="stepRow">
+                <Col xs={1}>
+                  <div className="numberCircle">
+                    <p>1</p>
+                  </div>
+                </Col>
+                <Col>
+                  <WrappedMessage id="step_01" messages={messages} />
+                </Col>
+              </Row>
+              <Row className="stepRow">
+                <Col xs={1}>
+                  <div className="numberCircle">
+                    <p>2</p>
+                  </div>
+                </Col>
+                <Col>
+                  <WrappedMessage id="step_02" messages={messages} />
+                </Col>
+              </Row>
+              <Row className="stepRow">
+                <Col xs={1} />
+                <Col>
+                  <a href={instanceLink}>
+                    <button
+                      id="manage_button"
+                      type="button"
+                      className="btn btn-primary"
+                    >
+                      <Row>
+                        <Col>
+                          <p>
+                            <WrappedMessage
+                              id="button_text"
+                              messages={messages}
+                            />
+                          </p>
+                        </Col>
+                        <Col xs={2}>
+                          <i className="fas fa-external-link-alt" />
+                        </Col>
+                      </Row>
+                    </button>
+                  </a>
+                </Col>
+              </Row>
+              <Row className="stepRow">
+                <Col xs={1} />
+                <Row>
+                  <WrappedMessage id="help_text" messages={messages} />
+                  &emsp;
+                  <Nav.Link
+                    className="support_link"
+                    onClick={() => window.open(CONTACT_US_LINK, '_blank')}
+                  >
+                    <WrappedMessage id="help_link" messages={messages} />
+                  </Nav.Link>
+                </Row>
+              </Row>
+            </ConsolePageCustomizationContainer>
+          </Container>
```

Comment:
This section can be heavily simplified down to this:
```suggestion
          <ConsolePageCustomizationContainer>
            <h2>
              <WrappedMessage id="title" messages={messages} />
            </h2>
            <WrappedMessage id="paragraph_01" messages={messages} />
            <div className="list-items">
              <Row>
                <Col xs={1}>
                  <div className="numberCircle">
                    <p>1</p>
                  </div>
                </Col>
                <Col>
                  <WrappedMessage id="step_01" messages={messages} />
                </Col>
              </Row>
              <Row>
                <Col xs={1}>
                  <div className="numberCircle">
                    <p>2</p>
                  </div>
                </Col>
                <Col>
                  <WrappedMessage id="step_02" messages={messages} />
                </Col>
              </Row>
            </div>

            <p>
              <a href={instanceLink}>
                <Button size="lg">
                  <WrappedMessage messages={messages} id="button_text" />
                  <i className="fas fa-external-link-alt fa-lg" />
                </Button>
              </a>
            </p>

            <h6>
              <WrappedMessage id="help_text" messages={messages} />
              <a className="support-link" href={CONTACT_US_LINK} target="_blank">
                <WrappedMessage id="help_link" messages={messages} />
              </a>
            </h6>
          </ConsolePageCustomizationContainer>
```

Notice the simpler structure and reusing styles that are already implemented in other pages.
The excessive use of `Row` components made the page harder to theme.

Reply:
It's true, this whole PR could've used some refactoring. Applied!